cmd/traffic/cmd/manager/service.go

```diff
@@ -443,7 +446,11 @@ func (s *service) PrepareIntercept(ctx context.Context, request *rpc.CreateInter
 	span := trace.SpanFromContext(ctx)
 	tracing.RecordInterceptSpec(span, request.InterceptSpec)
 
-	return s.state.PrepareIntercept(ctx, request)
+	replacePolicy := agentconfig.ReplacePolicyNever
+	if request.InterceptSpec.Replace {
+		replacePolicy = agentconfig.ReplacePolicyInactive
```

Comment:
Shouldn't this be ReplacePolicyActive?

Reply:
Nope; it's just the prepare call, the intercept isn't live yet. The CreateIntercept call will then switch it to Active. But this way Prepare is just used for the setup of the configmap, i.e. to ensure that it's impossible to later say --replace=false for instance

Reply:
If this were Active, I think it'd violate the API for PrepareIntercpept. After all, then it'd be possible to call Prepare without a subsequent Create and effectively bring down the service (since there's no listener for the agent to relay the request to)